app/controllers/admin/base_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::BaseController < ApplicationController
```

Comment:
So I think hound is complaining that we should instead be doing:

```ruby
  module Admin
    class BaseController < ApplicationController
     ..
    end
  end
```
I'll leave it on you if you want to switch to that instead.